docs/live_update_reference.md

```diff
@@ -14,6 +14,8 @@ The list of `LiveUpdateSteps` must be, in order:
 - 0 or more [`run`](api.html#api.run) steps
 - 0 or 1 [`restart_container`](api.html#api.restart_container) steps
 
+**NOTE**: your initial build will be a _full_ build---i.e., the specified Docker build or Custom Build. (This is because Live Update needs a running container to modify.) _Live Update will only run on your second build and thereafter._ Thus, your base build (Docker/Custom Build) should be sufficient to create your dev image, and should not rely on any `sync`'d files or `run` commands. (If this is functionality you need, [let us know](https://tilt.dev/contact).)
```

Comment:
The tone and syntax here (i.e., lots of **bold** and _italic_ and repeated parentheticals) feels like you're arguing with someone. That feels weird in the intro to a doc, where the user doesn't even know what you're talking about yet.

Maybe alter this to parallel the section below?

```
When you `tilt up`, your initial build will be a full build -- i.e., the specified Docker build or Custom build.

When a file changes:
   1) Live update will find the running container to modify
```
  2. 

maybe

Reply:
that's a really good point, will do